docs/preview/servicebus-queue-worker-template.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
Added a new `./docs/preview` place to set upcoming releases instead of changing the ones in `./docs/features`. This will be easier to manage both new features while still providing a place for the most recent published features.

Reply:
Can't we just add the next version already in the docs or add "Next release" in the badge name so that we have one file?

Just thinking out loud.

Reply:
Yes, but if we than actually release the "next version' we have to split it either way. And then we have to look into what has changed and not and all.

Reply:
Yes but then it's in the same file at least, now the diff just says there is a new file :)

Reply:
And why is the reason why we can't move it to the current files?

Reply:
Yes, I think there is maybe a little miscommunication here.
The reason why I created a whole new file, is that if version v0.3 is released, we just have to move the current `.\features\**` files to `.\v0.3\features\**` and move the `.\preview\features\**` to the "most recent published  features".
It allows us to fully change how the docs of this new version should look like. If we would place the changes in the same file, we would have possible duplicates of everything. Every title, every section, every code example... because there could be diff's between the already released version and the version we're working on. Moving them in sepearate files solves that problem.

Reply:
By duplicating it in the same file, you make it a lot harder for devs and consumers to look for the docs they need. Moving it to a seperate file solves that too.

Reply:
Ok, now I'v got it; sorry!